src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -70,6 +70,9 @@
         - name: --enable-aad
           type: bool
           short-summary: Enable managed AAD feature for cluster.
+        - name: --aad-enable-azure-rbac
```

Comment:
should be --enable-azure-rbac?

Reply:
As it is aad flag, i used aad as a prefix following the notation as used in aad-admin-group-object-ids and aad-tenant-id flags

Reply:
is that fine or should i use enable-azure-rbac?

Reply:
i'd think so. but please confirm with Jorge/Justin/Shashank

Reply:
@shashankbarsin, @palma21 , @jluk, what do think?

Reply:
Yes, enable-azure-rbac is good

Reply:
Changed to --enable-azure-rbac as discussed